### PR TITLE
fix broken column resize

### PR DESCRIPTION
**Pull request by jpdriver**
- [x] use .get() if it exists

### CODE REVIEW

File: src/ColumnMetrics.js [hunk 1]
```diff
@@ -99,7 +99,8 @@ function setDefferedColumnWidths(columns, unallocatedWidth, minColumnWidth) {
  * @param {number} width
  */
 function resizeColumn(metrics: ColumnMetricsType, index: number, width: number): ColumnMetricsType {
-  var column = metrics.columns[index];
+  // use get if we have it
+  var column = metrics.columns.get ? metrics.columns.get(index) : metrics.columns[index];
```
Comment (malonecj) [inline src/ColumnMetrics.js]: use ColumnUtils wrapper to get an individual column

File: src/Cell.js [hunk 2]
```diff
@@ -234,9 +234,6 @@ var Cell = React.createClass({
     // -> removing the class
     if(updateCellClass != null && updateCellClass != "") {
       this.getDOMNode().classList.remove(updateCellClass);
-      // -> triggering reflow /* The actual magic */
-      // without this it wouldn't work. Try uncommenting the line and the transition won't be retriggered.
```
Comment (bakesteve) [inline src/Cell.js]: so this is genuinely no longer needed?

File: package.json [hunk 3]
```diff
@@ -79,7 +79,7 @@
     "moment": "^2.8.1",
     "pretty-hrtime": "^0.2.1",
     "react-highlight": "^0.4.1",
-    "react-router": "^0.11.6",
+    "react-router": "0.11.6",
```
Comment (bakesteve) [inline package.json]: shouldnt this be a ~ or are we saying we ONLY want 0.11.6 (and NOT 0.11.7/8/9?)

File: src/ColumnMetrics.js [hunk 4]
```diff
@@ -99,7 +99,8 @@ function setDefferedColumnWidths(columns, unallocatedWidth, minColumnWidth) {
  * @param {number} width
  */
 function resizeColumn(metrics: ColumnMetricsType, index: number, width: number): ColumnMetricsType {
-  var column = metrics.columns[index];
```
Comment (bakesteve) [inline src/ColumnMetrics.js]: I dont think we should be doing this
inside columnMetrics, surely we already know if a column is an immutable or not?
Looks like the `ColumnMetricsType` definition needs to be update too - but assuming `metrics.columns: ImmutableList<Column>` and not `metrics.columns: ImmutableList<Column> | Aray<Column>`
obviously then means we need to trace back what is changing columns from ImmutableList -> Array?

**Pull request closed without merging**